src/commands/init.ts

```diff
@@ -46,48 +43,55 @@ export default async (positionals: Array<string>): Promise<void> => {
     }
 
     const packages = await getLocalPackages();
-    const doModelsExist = (await getModels(packages)).length > 0;
+
+    // This case should never happen, since we login before running the init command
```

Comment:
```suggestion
    // This case should never happen, since we log in before running the init command
```

As mentioned previously, "login" is a noun, so it's not suitable for this context. Please make sure that the comment is still under the max length for comments.